### ISSUE TITLE
Hypothesis: CI decreases time to release code

### ISSUE THREAD

**Issue by missaugustina** [BonnyCI/ci-plunder#23]
- [ ] Why is releasing code faster important? The answer to that question is the value we want to identify. "CI is valuable because it provides X by decreasing the time code releases become available to a project's users."
- [ ] How do we measure time to release?
(feel free to add items as they come up)
- [ ] Once the higher value has been identified, add a milestone "Measuring Value: X" where X is the value you've identified.

**Comment by missaugustina** [comment]
I updated the wiki with a couple of articles I found regarding this topic. One example that came up was the notion of "stability" and the ability to, for instance, release security-related fixes quickly or to respond to.

 * https://papers.ssrn.com/sol3/papers.cfm?abstract_id=2681906
 * https://papers.ssrn.com/sol3/papers.cfm?abstract_id=2681909

Another thing that's come up in my interviews is a sense of frustration when developers have to wait for the gate to finish a job. Even in my own experience, I've joked that "gating" is the new "compiling". Still nto sure what larger value that ties into but maybe there are some ideas there.
